library/advanced-search.en.md

```diff
@@ -0,0 +1,46 @@
+---
+Title: Advanced Search
+Category: Library
+Template: document
+Tags: support
+Date: 2017-07-20
+Slug: advanced-search
+Summary:
+Lang: en
+toc_run: true
+Untranslated: yes
+---
+
+<a name="advanced-search"></a>
+# Advanced search
+
+<a name="technical-background"></a>
+## Technical background
+
+opendata.swiss has a very powerful search engine, that can help you to find exactly the datasets you want.
```

Comment:
I think this statement is key. And should be reiterated in a page in the [Support](https://handbook.opendata.swiss/en/category/support) category which talks about how searching portals is a key instrument of data discovery.